ehrql/backends/tpp.py

```diff
@@ -378,9 +378,7 @@ def medications(self):
             CASE
                 WHEN CodeSystemId = 0 THEN ConceptId
             END AS snomedct_code,
-            CASE
-                WHEN CodeSystemId = 2 THEN ConceptId
-            END ctv3_code,
+            CTV3Code AS ctv3_code,
```

Comment:
question: Can you explain a little more of the context of what's happening here, and what problem it fixes, please? I know very little about OpenPROMPT.

Reply:
Is it something like "the existing data in the column is not what is expected, and we're having to pull that from another temporary column"?

Reply:
CTV3 codes were originally in `OpenPROMPT.CodedEvent_ID`. This was an error, so they were removed and this column became a FK to `CodedEvent.CodedEvent_ID`. It was assumed that the codes in `OpenPROMPT.ConceptId` -- which are a mix of CTV3 "Y" codes and SNOMED CT codes -- would be sufficient (`OpenPROMPT.CodeSystemId` is the flag that tells us the coding system). However, they were _not_ sufficient: there is an imperfect mapping between SNOMED CT codes and "regular" (non-"Y") CTV3 codes. So, all CTV3 codes were added to `OpenPROMPT.CTV3Code`. The above diff overcomes the imperfect mapping.

Reply:
Got it; thanks!